src/test/groovy/org/assertj/generator/gradle/parameter/TemplateChanges.groovy

```diff
@@ -270,7 +276,7 @@ class TemplateChanges {
                 main {
                     assertJ {
                         templates {
-                            methods { wholeNumberPrimitive = file('${contentPath}') }
+                            methods { wholeNumberPrimitive.file('${contentPath}') }
```

Comment:
we lost the nice functionality we had here of just `=` but I think it was misleading and confusing since passing in a `string` file name would cause it to treat it as a template -- not exactly "expected".